### PR TITLE
Added documentation and latest SQL database

### PR DESCRIPTION
**Pull request by jack-sandeman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,15 @@ New tests should be written where it makes sense. With Flywheel, the PHPUnit tes
 
 At this moment in time, the tests are not run automatically on public CI services.
 
+## Cypress Tests
+To run the tests you will need to: 
+
+- load the included SQL database, found in `/tests/cypress/fixtures/test-database.sql`
```
Comment (iamthomasbarrett) [inline README.md]: How do I load this SQL?

File: README.md [hunk 2]
```diff
@@ -134,6 +134,15 @@ New tests should be written where it makes sense. With Flywheel, the PHPUnit tes
...
+- `npx cypress run`
```
Comment (iamthomasbarrett) [inline README.md:141]: No dependencies to install?

Comment (jack-sandeman) [reply README.md:141]: It's covered by the npm install talked about on line 66

File: README.md (hunk 1 above)
Comment (jack-sandeman) [reply README.md]: I could add a comment saying open your SQL database editor of choice, but past that it would change depending on what you choose

Comment (iamthomasbarrett) [reply README.md]: Is it a complete WP database?

Comment (iamthomasbarrett) [reply README.md]: Just trying to give a sense of context to it.

Comment (jack-sandeman) [reply README.md]: I think so yes

Comment (iamthomasbarrett) [reply README.md]: I would put that in the comment. 

"Replace your existing WP database with this test snapshot"

**Review by iamthomasbarrett — APPROVED** [review]

**Pull request merged**